sample/spec/lib/load_sample_spec.rb

```diff
@@ -9,4 +10,14 @@
       SpreeSample::Engine.load_samples
     }.to output.to_stdout
   end
+
+  it "has db:seed as a prerequisite" do
+    task = Rake::Task["spree_sample:load"]
+    seed_task = Rake::Task["db:seed"]
+    expect(task.prerequisite_tasks).to include(seed_task)
+  end
+
+  before do
+    Rails.application.load_tasks
```

Comment:
If that's only a requirement for the new test, we could have it within the `it` block, so it doesn't affect other tests in the file.